packages/block-library/src/navigation-link/style.scss

```diff
@@ -28,6 +28,10 @@
 
 // Styles for submenu flyout
 .has-child {
+	$navigation-vertical-padding: $grid-unit-10 * 0.75;
```

Comment:
Hmm, why do we need this 🤔 ? Is it used anywhere?

Reply:
The variable is actually no longer needed. It was also removed in the previous PR.

I have now removed it too and tested everything again. So far everything looks good. Thanks!